e2e/imgbuild/imgbuild.go

```diff
@@ -2114,18 +2387,20 @@ CMD /bin/true
 			whileLoggedIn: true,
 			expectExit:    0,
 		},
-		{
-			name:          "privimg df logged out",
-			args:          []string{"-F", "--oci", "--no-https", "--disable-cache", "./my_image_file.oci.sif", dockerfile},
-			whileLoggedIn: false,
-			expectExit:    255,
-		},
-		{
-			name:          "privimg df logged in",
-			args:          []string{"-F", "--oci", "--no-https", "--disable-cache", "./my_image_file.oci.sif", dockerfile},
-			whileLoggedIn: true,
-			expectExit:    0,
-		},
+		// The following is disabled because it is now subsumed under
```

Comment:
Just remove this code. Significant commented chunks inevitably get out of date eventually, and can then confuse.

You can keep the removal in a separate commit with a message explaining it if you want to record the context of the removal.